### ISSUE TITLE
About Epochs for Effective Color Transfer in Model Training

### ISSUE THREAD

**Issue by ChesserZS** [Wp-Zhang/Deep-Color-Transfer#44]
Thank you for your training code and dataset. I have been using your dataset and training code for training, and it took a few days to train the model up to 30 epochs. However, the train loss and val loss have become difficult to decrease, and the color transfer effect is still quite poor. 
Could you please let me know around how many epochs it took for you to start seeing good color transfer results? And how many epochs did you train until convergence?

**Comment by Wp-Zhang** [comment]
We didn't train the model from scratch, instead we used the trained model from the author as a teacher model for knowledge distillation. For more training details, please refer to the Github Page of this project where loss curves are included.
Actually, we encountered the same issue as yours when we made the attempt of training the model from scatch at the very beginning of this project. I personally believe it was because of the missing of some implementation details and training tips that are not mentioned in the original paper and the according Github repo.
